libkbfs/kbfs_ops.go

```diff
@@ -247,6 +246,12 @@ func (fs *KBFSOpsStandard) GetOrCreateRootNode(
 			return nil, EntryInfo{}, err
 		}
 	}
+
+	if !create && md.data.Dir.Type != Dir {
+		kbpki := fs.config.KBPKI()
+		return nil, EntryInfo{}, identifyHandle(ctx, kbpki, kbpki, h)
```

Comment:
That is required for TestReaddirPublicFailedIdentifyViaOSCall. Is there a better way to do it?